### PR TITLE
Add past changes to ChangeLog.

### LINKED ISSUE #155: Update change log
Unfortunately, `changelog.rst` is very out of date...

### PR DESCRIPTION
**Pull request by sobolevnrm**
Fixes #155.

### CODE REVIEW

**Comment by sobolevnrm** [comment]
Fixes #155

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -8,10 +8,31 @@ Current version
...
+Fixes
+-----
+
+* Refactor and fix ``MolecularContainer.get_pi()``
+  (`#148 <https://github.com/jensengroup/propka/pull/148>`_)
+
+* Rename vanadium from ``Va`` to ``V``
+  (`#141 <https://github.com/jensengroup/propka/pull/141>`_)
+
+* Fix rounding issues
```
Comment (speleo3) [inline docs/source/changelog.rst:27]: I suggest to be a bit more specific: "Fix rounding issues in folding profile reporting"

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -8,10 +8,31 @@ Current version
...
 
-v3.4.0
-======
+* Add context manager for ``open_file_for_reading``
+  (`#133 <https://github.com/jensengroup/propka/pull/133>`_)
+
+Fixes
+-----
+
+* Refactor and fix ``MolecularContainer.get_pi()``
```
Comment (speleo3) [inline docs/source/changelog.rst:21]: The precision fix was the most relevant change, so I suggest:
```
* Fix precision of ``MolecularContainer.get_pi()``
```

**Review by speleo3 — CHANGES_REQUESTED** [review]
Thanks a lot, this looks good. My only suggestion is to be more specific in the list of fixes.

File: docs/source/changelog.rst (hunk 2 above)
Comment (sobolevnrm) [reply docs/source/changelog.rst]: Fixed in 0de6387b6909558e94ed690412d91872dd579f70.

File: docs/source/changelog.rst (hunk 1 above)
Comment (sobolevnrm) [reply docs/source/changelog.rst]: Fixed in c74e7cda665637f334881f3e2434dca7d365b818.

**Pull request merged**